### PR TITLE
Add scroll-driven-animations.dist.yml

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

File: feature-group-definitions/scroll-driven-animations.dist.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  - css.properties.animation-timeline.scroll
+  - css.properties.animation-timeline.view
+  # https://crbug.com/337580010 tracks this bug, but setting the timeline via
+  # the animation shorthand is not necessary to use the feature.
+  # - css.properties.animation.animation-timeline_included
+  - css.properties.scroll-timeline
+  - css.properties.scroll-timeline-axis
+  - css.properties.scroll-timeline-name
+  # Later addition (Chrome 116)
+  # - css.properties.timeline-scope
```
Comment (ddbeck) [inline feature-group-definitions/scroll-driven-animations.dist.yml:36]: I would like to be convinced on this. What's the case for pulling this one out, when they landed just four weeks apart?

Comment (foolip) [reply feature-group-definitions/scroll-driven-animations.dist.yml:36]: https://chromestatus.com/feature/6752840701706240 says 115 and I expect caniuse would say 115 too if it was added. One bit of an API coming a release later is pretty common I think. Unless it's a terrible accident and the feature is unusable without that bit, I think https://github.com/web-platform-dx/web-features/issues/979 is the way to handle such cases.

**Review by ddbeck — APPROVED** [review]

File: feature-group-definitions/scroll-driven-animations.dist.yml (hunk 1 above)
Comment (captainbrosset) [reply feature-group-definitions/scroll-driven-animations.dist.yml:36]: Maybe @bramus can share some insights about whether the `timeline-scope` property is a vital part of scroll-driven-animations, or whether the feature is mostly usable without it.

Comment (bramus) [reply feature-group-definitions/scroll-driven-animations.dist.yml:36]: `timeline-scope` is there for when want to animate an element based on a scroller **that is not a parent of the animated element**.

In most cases though, the animation subject _is_ a child of the scroller – I’ve only used `timeline-scope` a few times in my demos.

**Pull request merged**